app/models/User.coffee

```diff
@@ -213,6 +212,12 @@ module.exports = class User extends CocoModel
     return 0 unless numVideos > 0
     return me.get('testGroupNumber') % numVideos
 
+  getYearSubscriptionGroup: ->
+    return @yearSubscriptionGroup if @yearSubscriptionGroup
```

Comment:
I'm not sure memoizing this is best — we've had a lot of race condition type things where something will look at the instance variable instead of using the function that may or may not have been called yet. The function is basically free in terms of CPU cycles.